hardware/fastcomtec/fastcomtecp7887.py

```diff
@@ -239,9 +239,10 @@ def get_constraints(self):
         constraints['hardware_binwidth_list'] = list(self.minimal_binwidth * (2 ** np.array(
                                                      np.linspace(0,24,25))))
         constraints['max_sweep_len'] = 6.8
+        constraints['max_bins'] = 6.8/0.25e-9
         return constraints
 
-    def configure(self, bin_width_s, record_length_s, number_of_gates=0, filename=None):
+    def configure(self, bin_width_s, record_length_s, number_of_gates=1, filename=None):
```

Comment:
Is change in default value for `number_of_gates` really necessary? I think this could lead to errors because the method will now return 1 as number of gates by default, even though it is recognized by the logic as ungated counter.